### PR TITLE
[Test] Add test for Mill integration

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -72,6 +73,10 @@ jobs:
           ]
         include:
           - type: gradle-mac
+            command: bin/test.sh 'slow/testOnly -- tests.mill.*'
+            name: Mill Windows integration
+            os: windows-latest
+          - type: mill-windows
```
Comment (kpbochenek) [inline .github/workflows/ci.yml:79]: seems like should be other way around
gradle-mac => -- tests.gradle.*'
mill-windows => -- tests.mill.*'

Comment (kpbochenek) [reply .github/workflows/ci.yml:79]: unless you don't plan to merge that, then nvm :P

Comment (tgodzik) [reply .github/workflows/ci.yml:79]: Haha, yeah,  it's just for testing though.

**Comment by tgodzik** [comment]
Doesn't seem it's detecting the issue. Will need to work on it using windows, time to see if my dual boot stil works :sweat_smile:

**Pull request closed without merging**